### ISSUE TITLE
Addon not working on Ember 3.11

### ISSUE THREAD

**Issue by ehubbell** [mminkoff/ember-filestack#16]
We recently updated our application to Ember 3.11 and this addon stopped working for us. We kept getting an error message that contained `chunk response` something. 

I then went to install `filestack-js` and just use the NPM module directly and, after install, the addon started working for us again. Pretty sure this addon needs to update it's dependency on `filestack-js` from  `"filestack-js": "^3.8.0"` to `"filestack-js": "^3.15.0"` for later versions of Ember.

**Comment by miguelcobain** [comment]
Seems like master already has these updates. @mminkoff let us know if you need something from us to get it released.

**Comment by mminkoff** [comment]
Thanks @miguelcobain - yes, it's all done except I'm hitting this problem when running ember try: https://github.com/ember-cli/babel-plugin-ember-modules-api-polyfill/issues/64, which hopefully will be resolved soon.

@ehubbell what's in master should be safe to use if you'd like.  I just don't want to publish it with failing tests.

**Comment by rwjblue** [comment]
ember-cli-babel@7.20.1 was just released, and should fix the underlying issue referenced in https://github.com/ember-cli/babel-plugin-ember-modules-api-polyfill/issues/64.

https://github.com/babel/ember-cli-babel/releases/tag/v7.20.1

**Comment by mminkoff** [comment]
ember-filestack@3.02 was just published.  It includes updating filestack.js to 3.15 and ember to 3.18.  Thanks @ehubbell for reporting this, and sorry it took a few days to get it published.

**Comment by ehubbell** [comment]
@mminkoff  I just updated my project to use the latest versions of `ember-filestack` and `ember-cli-babel` and I'm still running into this issue. Here are the relevant versions:

```
node: 11.2
ember: 3.11
ember-filestack: 3.1.1
filestack-js: 3.15
ember-cli-babel:  7.22.1
```

Then when I use `ember-filestack` as outlined in the `readme` I get the following error:

```
vendor-suffix.js:1 Uncaught (in promise) Error: Loading chunk 0 failed.
(error: http://localhost:4200/assets/chunk.b875d74302a42040ef02.js)
    at HTMLScriptElement.onScriptComplete (vendor-suffix.js:1)
```

Any idea what is happening here? I also tried with `ember-cli-babel: 7.20.1` per @rwjblue post above to no avail.

**Comment by mminkoff** [comment]
@ehubbell Can you try updating ember-auto-import to 1.6.0?  This seems to be a webpack issue, which is used by ember-auto-import.

**Comment by ehubbell** [comment]
@mminkoff Good call. That fixed the issue 😄

**Comment by mminkoff** [comment]
Great!  I’ll try to get an update out tomorrow.

**Comment by mminkoff** [comment]
@ehubbell I just published v3.1.2 that now hopefully has everything you need.  Thanks for reporting the issue and trying out upgrading ember-auto-import.
